config/application.rb

```diff
@@ -16,6 +16,18 @@ class Application < Rails::Application
     # Common ones are `templates`, `generators`, or `middleware`, for example.
     config.autoload_lib(ignore: %w(assets tasks))
 
+    # config/initializers/locale.rb
+
+    # Where the I18n library should search for translation files
+    I18n.load_path += Dir[Rails.root.join('lib', 'locale', '*.{rb,yml}')]
```

Comment:
with this approach now is possible to use the structure in my last request. 👀

Reply:
that was done in a preview commit :+1: